.eslintrc.json

```diff
@@ -45,6 +45,12 @@
         "no-prototype-builtins": "off",
         "local-rules/no-budibase-imports": "error"
       }
+    },
+    {
+      "files": ["packages/builder/**/*"],
```

Comment:
We should probably apply this to the client library as well!

Reply:
I have actually already pushed an update with that, but github hasn't processed it yet. Super weird. 
![Uploading Screenshot 2024-01-31 at 10.56.40.png…]()